using_redcap_api/using_redcap_api.md

```diff
@@ -0,0 +1,436 @@
+
+![List of applications which includes the "API" and "API Playground" applications](media/api_api_playground.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+### API Token
+
+So now you have the **right** to use the API, but you can't start using it just yet. You have to generate an API token.  Click on the "API" application in the left side menu and in the main part of your screen, click on the button to "Generate API Token."  Depending on your institution's rules, you might automatically be granted a token immediately, or it might be generated manually by an administrator a few minutes or hours later.
+
+![Text and clickable button for obtaining an API token](media/obtain_api_token.png)<!-- style = "max-width:600px;" -->
+
+The API token is unique to the combination of user and project. It's a code that allows access only to the data in a single project and only the data that the person who generated the API token is allowed to see.  If you have three different projects, your token is different for each one. Importantly, if you feel like you may have accidentally given your API token away, it's a good idea to regenerate it, which is a single-click operation. It makes your old token invalid and creates a new token.
+
+Soon, your API token will show up in that same "API" Application. It will look something like this (note, I've obscured my token – sharing this in a post like this one would be a terrible idea!).
+
+![Text that includes an obscured API token](media/your_api_token.png)<!-- style = "max-width:600px;" -->
+
+You'll use this token when you request data in your R or Python script.
+
+### Quiz: REDCap API Token
+
+Which of the following is true of a REDCap API Token?  Select all that apply!
+
+[[ ]] As soon as you enable API Export in User Rights, an API token is generated for you
+[[X]] A REDCap API token is unique to you (no one else on a project has the same API token)
+[[X]] A REDCap API token is unique to the project (none of your other projects will give you the same API token)
+[[ ]] The REDCap API token is permanent and unchanging
+[[X]] Before generating an API token, you must have API rights given to you by someone who can change User Rights
+[[ ]] The REDCap API changes daily for added security
+[[?]] There are multiple correct answers!
+***************
+
+<div class = "answer">
+You can generate a REDCap API token once you have been given API rights.  Once you have API rights, your token isn't automatically generated, but it's an easy single click to get your token when you're ready.  Your token is unique both to you and to the project -- it's a unique key that will only work for you, and only work for the project it was generated for.  The REDCap API token doesn't change daily, but you can change it if you feel like it may have been accidentally shared to another person.
```

Comment:
```suggestion
You can generate a REDCap API token once you have been given API rights.  Once you have API rights, your token isn't automatically generated, but it's an easy single click to get your token when you're ready.  Your token is unique both to you and to the project -- it's a unique key that is only meant for you, and only for the project it was generated for.  That said, anyone who has that key can access the data, so you should not share it with anyone. The REDCap API token doesn't change daily, but you can change it if you feel like it may have been accidentally shared to another person.
```